rust/lb-operator/src/main.rs

```diff
@@ -0,0 +1,99 @@
+use std::{os::unix::prelude::FileTypeExt, path::PathBuf};
+
+use clap::Parser;
+use csi_server::{
+    controller::LbOperatorController, identity::LbOperatorIdentity, node::LbOperatorNode,
+};
+use futures::{pin_mut, FutureExt, TryStreamExt};
+use grpc::csi::v1::{
+    controller_server::ControllerServer, identity_server::IdentityServer, node_server::NodeServer,
+};
+use stackable_operator::{kube::CustomResourceExt, logging::TracingTarget};
+use tokio::signal::unix::{signal, SignalKind};
+use tokio_stream::wrappers::UnixListenerStream;
+use tonic::transport::Server;
+use utils::{uds_bind_private, TonicUnixStream};
+
+use crate::crd::{LoadBalancer, LoadBalancerClass};
+
+mod crd;
+mod csi_server;
+mod grpc;
+mod lb_controller;
+mod utils;
+
+#[derive(clap::Parser)]
+#[clap(author, version)]
```

Comment:
Use the built_info crate hereto stay consistent with the other operators?

Reply:
Myeah, I'm trying to minimize the usage of `built` since it (by default) relies on a bunch of environment variables that Cargo sets but Nix doesn't. It works in opt-in-per-feature mode, but then it seems pointless to opt into a feature that `clap` already provides.